<launch>

  <arg name="svo_file" default=""/>
  <arg name="zed_id" default="0"/>

  <group ns="zed">
    <include file="$(find zed_wrapper)/launch/custom_zed_camera.launch">
      <arg name="svo_file" value="$(arg svo_file)" />
      <arg name="zed_id" value="$(arg zed_id)" />
    </include>
  </group>

  <include file="$(zenith_localization)/launch/Zed16h5TagSingleI.launch">

  <node name="zed_obsticale_detector" pkg="zenith_obstacle_detector" type="zed_obsticale_detector"/>
  <node name="driver" pkg="zenith_sabertooth_driver" type="driver.py"/>
  <node name="planner" pkg="zenith_path_planner" type="planner.py"/>
  
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">

</launch>
